main.nf

```diff
@@ -791,8 +816,7 @@ process proteomicslfq {
     
     input:
      file mzmls from mzmls_plfq.map{it[1]}.toSortedList({ a, b -> b.baseName <=> a.baseName })
-     file id_files from id_files_idx_feat_perc_fdr_filter_switched
-         .mix(id_files_idx_ForIDPEP_fdr_switch_idpep_switch_filter_switch)
```

Comment:
Missing: Mixing in percolator and fdr IDs when phospho_rescoring is not set

Reply:
yes, should be easy. just a bit ugly. but that is how it is ;)

Reply:
ok ill adapt this part then